packages/web-react/src/components/Alert/README.md

```diff
@@ -17,13 +17,17 @@ import { Alert } from '@lmc-eu/spirit-web-react/components';
 
 ## Default Icons according to Color Variant
 
-| Color name    | Icon name     |
-| ------------- | ------------- |
-| `danger`      | `warning`     |
-| `default`     | `info`        |
-| `informative` | `info`        |
-| `success`     | `check-plain` |
-| `warning`     | `warning`     |
+| Color name    | Icon name            |
+| ------------- | -------------------- |
+| `danger`      | `warning`/`danger`\* |
+| `default`     | `info`               |
+| `informative` | `info`               |
+| `success`     | `check-plain`        |
+| `warning`     | `warning`            |
+
+(\*) The `warning` icon for the `danger` color has been marked as [deprecated][readme-deprecated].
```

Comment:
FYI @literat Storybook markdown reader doesn't like implicit reference links (aka those without second brackets), so I try to avoid them because it won't make them links and print them just as text with brackets.

Reply:
Ok, thanks for the fix!